### PR TITLE
230304 invoice line fields

### PR DESCRIPTION
**Pull request by JasmeetKaur-Defra**

### CODE REVIEW

File: EST.MIT.Web/Repo/ReferenceDataRepository.cs [hunk 1]
```diff
@@ -65,6 +89,30 @@ private async Task<HttpResponseMessage> GetPaymentTypesList(string? InvoiceType,
         return response;
     }
 
+    private async Task<HttpResponseMessage> GetAccountsList
+       (string InvoiceType, string Organisation, string Scheme, string PaymentType)
```
Comment (andygibbonsdefra) [inline EST.MIT.Web/Repo/ReferenceDataRepository.cs:93]: Should the Scheme variable names (that are passed as params to the RefData fetches the the five dropdown info data lists) be named SchemeType?

File: EST.MIT.Web/Repo/ReferenceDataRepository.cs [hunk 2]
```diff
@@ -65,6 +89,30 @@ private async Task<HttpResponseMessage> GetPaymentTypesList(string? InvoiceType,
...
+    {
+        var client = _clientFactory.CreateClient("ReferenceDataAPI");
+
+        var response = await client.GetAsync($"/accounts/{InvoiceType}/{Organisation}/{Scheme}/{PaymentType}");
```
Comment (andygibbonsdefra) [inline EST.MIT.Web/Repo/ReferenceDataRepository.cs:97]: So this should maybe be SchemeType and not Scheme?

Comment (JasmeetKaur-Defra) [reply EST.MIT.Web/Repo/ReferenceDataRepository.cs]: Changed the name to SchemeType

File: EST.MIT.Web/Repo/ReferenceDataRepository.cs (hunk 1 above)
Comment (JasmeetKaur-Defra) [reply EST.MIT.Web/Repo/ReferenceDataRepository.cs]: Done, changed the variable name to SchemeType

**Review by andygibbonsdefra — APPROVED** [review]

**Pull request merged**